.github/workflows/test.yml

```diff
@@ -0,0 +1,135 @@
+name: Test
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: yarn
+        cache-dependency-path: "**/yarn.lock"
+    - run: yarn install
+    - run: yarn test:health
+    - run: yarn flow
+    - run: yarn typescript
+    - run: yarn lint
+    - run: yarn test:unit
+
+  build:
```

Comment:
We should also build it with node14 and node16 for propper testing in matrix like style